release-notes/6.0/known-issues.md

```diff
@@ -38,6 +38,12 @@ When opening a project in Visual Studio, you may get an error similar to the fol
 
 > The project file cannot be opened. The NuGet-based SDK resolver failed to run because NuGet assemblies could not be located. Check your installation of MSBuild or set the environment variable “MSBUILD_NUGET_PATH” to the folder that contains the required NuGet assemblies. Could not find file ‘C:\Program Files\dotnet\sdk-manifests\6.0.100\Microsoft.NET.Workload.Android\WorkloadManifest.json’.
 
+When opening a project in Visual Studio for Mac, you may get an error similar to the following:
```

Comment:
Is this applicable to Preview 5 or Preview 6?

Reply:
Both, as I understand it. It applies to any upgrade from Preview 4. 

(Adding in @mrward who helped me find this.)